test/parallel/test-async-wrap-uid.js

```diff
@@ -6,15 +6,14 @@ const assert = require('assert');
 const async_wrap = process.binding('async_wrap');
 
 const storage = new Map();
-async_wrap.setupHooks({ init, pre, post, destroy });
```

Comment:
no. it just makes triggering destroy() before exit a pain. and haven't figured out a good way to test it yet.